users_app/pipeline/pipeline.py

```diff
@@ -1,24 +1,56 @@
+from abc import abstractmethod
+
 from django.contrib.auth.models import Group
 
 from users_app import ALLOWED_AP_ACCOUNTS
 
 
-def add_user_to_group(is_new, user, *args, **kwargs):
-    if is_new:
-        if user.email in ALLOWED_AP_ACCOUNTS:
-            ap_admin_group = Group.objects.get(name='ap_admin')
-            ap_manager_group = Group.objects.get(name='ap_manager')
-            user.groups.add(ap_admin_group, ap_manager_group)
+def get_ap_allowed_accounts():
+    return ALLOWED_AP_ACCOUNTS
```

Comment:
I needed to patch that constant, but when I tried I couldn't get the value of the list, but also a MagicMock object, so the test failed, because I can't access to the value.
But if I patch the function I can get the value of the list.
That's the reason why I created it.